tests/pass_by_ref_test.py

```diff
@@ -193,11 +209,21 @@ def test_old():
 
 
 if not OLD_CODE_ONLY:
-    NEW_CASES = (sub_logcat_dynamic, swapper, wilt_the_stilt, fac_by_ref, sub_mixed)
+    NEW_CASES = (
+        sub_logcat_dynamic,
+        swapper,
+        wilt_the_stilt,
+        fac_by_ref,
+        sub_mixed,
+        user_guide_docs_dynamic_scratch_var_demo,
+    )
 
     def test_swapper():
         compile_and_save(swapper)
 
+    def test_user_guide_docs_dynamic_scratch_var_demo():
```

Comment:
This test is redundant given the fact that you added `user_guide_docs_dynamic_scratch_var_demo` to `NEW_CASES`. (`assert_new_v_old` calls `compile_and_save` as a subroutine)

Reply:
@tzaffi Agree - I removed the redundancy in https://github.com/algorand/pyteal/pull/226/commits/2bebeb30ba17f8b36f5dd0403a0ba5b48661c52a.